admin_guide/manage_images.adoc

```diff
@@ -0,0 +1,83 @@
+[[admin-guide-manage-images]]
+= Managing Nodes
```

Comment:
@soltysh Also, I think you meant the title and anchor to reflect Monitoring Images too.